src/tasks/shop-admin/Settings/ConfigSalesChannel.ts

```diff
@@ -0,0 +1,32 @@
+import { test as base, expect } from '@playwright/test';
+import type { Task } from '../../../types/Task';
+import type { FixtureTypes} from '../../../types/FixtureTypes';
+
+export const ConfigSalesChannel = base.extend<{ ConfigSalesChannel: Task }, FixtureTypes>({
```

Comment:
here you have to run the `clearCache` function because it is `systemConfig` related.

Reply:
It works without cache clear.